lib/mastodon/maintenance_cli.rb

```diff
@@ -175,8 +175,175 @@ def fix_duplicates
       @prompt.say 'Finished!'
     end
 
+    MEDIA_TYPES = %w(accounts custom_emojis media_attachments).freeze
+
+    option :domain, type: :string, required: true, desc: 'Target server domain'
+    option :only, type: :array, enum: MEDIA_TYPES, desc: 'Only process these media types'
+    option :force, type: :boolean, desc: 'Skip new remote_url check'
+    desc 'fix-remote-url FROM TO', 'Fix remote_url of existing media'
+    long_desc <<-DESC
+      Fix the remote_url of the existing media when the remote media server is changed.
+
+      It will automatically check if the media can be obtained from the changed URL. To avoid this, specify the --force option.
+    DESC
+    def fix_remote_url(from, to)
+      @prompt = TTY::Prompt.new
+
+      @from   = from
+      @to     = to
+      @domain = options[:domain]
+      @types  = options[:only] || MEDIA_TYPES
+      @force  = options[:force]
+
+      @prompt.say "Start fixing the remote_url of #{@types.join ', '} on the remote server \"#{@domain}\" from \"#{@from}\" to \"#{@to}\"."
+      raise 'User canceled' unless @prompt.yes?('Continue?')
```

Comment:
I followed the advice of RuboCop and DeepSource, but what is the really desirable way to write it? (exit(1) is used in many places)

Reply:
Doesn't raising an exception only make sense when a call further up the stack could catch it? If the user doesn't want to fix what's shown exit seems just as good here as elsewhere in this file.

Reply:
Change it to exit(1), which is the same as elsewhere.